.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
   global:
     - DOCKER_COMPOSE_VERSION=1.25.5
     - COMPOSE_PROJECT_NAME=pupperware
+    - PRELOAD_CERTS=1
+    - PUPPETSERVER_VERSION=edge
```

Comment:
This is necessary until the next actual release version of puppetserver 7, which will then become "latest"